generators/server/files.js

```diff
@@ -925,7 +925,7 @@ const serverFiles = {
             ]
         },
         {
-            condition: generator => generator.applicationType === 'monolith' && generator.authenticationType !== 'oauth2',
+            condition: generator => !shouldSkipUserManagement(generator) && generator.authenticationType !== 'oauth2',
```

Comment:
This uses the same condition for generating `DomainUserDetailsServiceIntTest` as it does for generating `DomainUserDetailsService`